### PR TITLE
bugfix at nginx server detection

### PR DESCRIPTION
**Pull request by adibalcan**

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -4628,7 +4628,7 @@
 				22
 			],
 			"headers": {
-				"Server": "nginx(?:/([\\d.]+))?\\;version:\\1"
```
Comment (q--) [inline src/apps.json]: Why should Wappalyzer stop detecting the NGINX version?

Comment (adibalcan) [inline src/apps.json]: last nginx version haven't version in server header

File: src/apps.json [hunk 2]
```diff
@@ -4628,7 +4628,7 @@
...
+				"Server": "nginx.*"
```
Comment (gadcam) [inline src/apps.json]: Both patterns will match the same thing as .* is "optional".

File: src/apps.json (hunk 1 above)
Comment (gadcam) [inline src/apps.json]: The <code>?</code> is making it optional.

Comment (q--) [inline src/apps.json]: @adibalcan The version is still shown for version 1.9.8 at http://nginx.org/. Also, as @gadcam pointed out, your change doesn't improve detection; [`?`](http://www.regular-expressions.info/optional.html) after the group makes it optional. Furthermore, even if a current version of the application doesn't reveal the version number, that's still no reason to kill version detection for older versions.

**Pull request closed without merging**